features/bevragen/persoon-beperkt/overlijden/dev/fields-gba.feature

```diff
@@ -0,0 +1,69 @@
+#language: nl
+
+@gba
+Functionaliteit: Persoon: overlijden velden vragen met fields
+
+  Abstract Scenario: 'datum overlijden (08.10)' wordt gevraagd met field pad '<fields>'
+    Gegeven de persoon met burgerservicenummer '000000152' heeft de volgende gegevens
+    | naam                  | waarde   |
+    | geslachtsnaam (02.40) | Maassen  |
+    | geboortedatum (03.10) | 19830526 |
+    En de persoon heeft de volgende 'overlijden' gegevens
+    | naam                     | waarde   |
+    | datum overlijden (08.10) | 20020701 |
+    Als gba personen wordt gezocht met de volgende parameters
+    | naam                       | waarde                              |
+    | type                       | ZoekMetGeslachtsnaamEnGeboortedatum |
+    | geslachtsnaam              | Maassen                             |
+    | geboortedatum              | 1983-05-26                          |
+    | inclusiefOverledenPersonen | true                                |
+    | fields                     | <fields>                            |
+    Dan heeft de response een persoon met de volgende 'overlijden' gegevens
+    | naam  | waarde   |
+    | datum | 20020701 |
+
+    Voorbeelden:
+    | fields                        |
+    | overlijden                    |
+    | overlijden.indicatieOverleden |
+    | overlijden.datum              |
```

Comment:
mag je bij zoeken vragen om overlijden.datum? Die zit toch niet in de PersoonBeperkt?

Reply:
klopt. Overblijfsel van copy paste actie vanuit persoon. Het is niet opgevallen omdat dit nu geen foutmelding geeft